projects/core/utils/miscellaneous/hint-is-obscured.ts

```diff
@@ -0,0 +1,13 @@
+import {tuiGetElementObscures} from '@taiga-ui/cdk';
+
+/**
+ * @internal
+ */
+export function tuiHintIsObscured(
+    el: HTMLElement,
+    exceptSelector = `tui-hints-host`,
+): boolean {
+    const obscured = tuiGetElementObscures(el);
+
+    return !!obscured?.length && obscured.some(el => !el.closest(exceptSelector));
```

Comment:
Would `!!tuiGetElementObscures(el)?..some(el => !el.closest(exceptSelector))` be the same? 🤔